src/components/hierarchy-view/hierarchy.tsx

```diff
@@ -44,7 +45,7 @@ export const Hierarchy = (props: IProps) => {
     };
 
     return (
-      <>
+      <div className={css.hierarchyWrapper} onClick={handleShowComponent}>
```

Comment:
Do we need these inner `onClick` handlers here and on the table view since you also added it to the outer app?

Reply:
There is no outer app. The select view, heirarchy view, and the nested table view replace each other as the app so I had to add the onClick handler to each of the views.